app/assets/stylesheets/container_topology.css

```diff
@@ -54,7 +54,7 @@ kubernetes-topology-graph {
 }
 
 .search-topology .form-control {
-    font-size: 20px;
+    font-size: 16px;
 }
 
```

Comment:
@serenamarie125, does the search have to be so tiny? (12px)
I wanted to make it larger to be pleasant to the eye, but @epwinchell pointed me to the pf standard that is 12px